Jenkinsfile_k8s

```diff
@@ -12,14 +14,14 @@ pipeline {
     stage('Updatecli DIFF') {
       steps {
         catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
-          updatecli(action: 'diff', updatecliDockerImage: 'jenkinsciinfra/hashicorp-tools:0.3.7') //keep version uptodate
+          updatecli(action: 'diff', updatecliDockerImage: '$dockerImage')
```

Comment:
```suggestion
          updatecli(action: 'diff', updatecliDockerImage: dockerImage)
```

In [Groovy](https://groovy-lang.org/syntax.html) , the use of simple quotes means "litteral string" so your variable won't be interpolated.

Also, using the `${VAR}` requires mandatory brackets AND is only possible in interpolated strings (e.g. strings with double quotes).


But using a variable identifier is always valid, hence the suggestion.